platform-codingcontest-sync/build.gradle

```diff
@@ -1,3 +1,4 @@
 dependencies {
     compile project(':platform-server-kernel')
+    compile "com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.7.4"
```

Comment:
remove version - `:2.7.4`, it will be automatically managed, and it would be then `:2.6.6` if this is acceptable